src/element/textWysiwyg.test.tsx

```diff
@@ -16,7 +16,7 @@ describe("textWysiwyg", () => {
 
     const element = UI.createElement("text");
 
-    new Pointer("mouse").clickOn(element);
+    new Pointer("mouse").doubleClickOn(element);
```

Comment:
Unclear how this test ever worked and why exactly it stopped working now, but it only shows how little of our tests actually test what we thing they test. Headless testing with a mocked DOM is simply not great.